src/occ_gordon/occ_gordon.cpp

```diff
@@ -20,6 +22,30 @@ namespace occ_gordon
 Handle(Geom_BSplineSurface) interpolate_curve_network(const std::vector<Handle (Geom_Curve)> &ucurves,
                                                       const std::vector<Handle (Geom_Curve)> &vcurves,
                                                       double tolerance)
+{
+    std::vector<Handle(Geom_BSplineCurve)> ucurves_bsplines, vcurves_bsplines;
+
+    ucurves_bsplines.reserve(ucurves.size());
+    vcurves_bsplines.reserve(vcurves.size());
+
+    try {
+		for (const auto& profile : ucurves) {
+			ucurves_bsplines.push_back(GeomConvert::CurveToBSplineCurve(profile));
+		}
+		for (const auto& guide : vcurves) {
+            ucurves_bsplines.push_back(GeomConvert::CurveToBSplineCurve(guide));
```

Comment:
Please don't use tabs for indentation.

```suggestion
        for (const auto& profile : ucurves) {
            ucurves_bsplines.push_back(GeomConvert::CurveToBSplineCurve(profile));
        }
        for (const auto& guide : vcurves) {
            ucurves_bsplines.push_back(GeomConvert::CurveToBSplineCurve(guide));
```

Reply:
That's weird - VS setup to replace tabs with spaces, not sure what happened here. Will fix.